web/services/getProjectsService.php

```diff
@@ -85,26 +94,31 @@
         break;
     }
 
-    $projects = ProjectsFacade::GetAllProjects($login, $active, $order,$description, $filterStartDate, $filterEndDate,
-        $activation, $areaId, $type, $cname);
-
+    $projects = ProjectsFacade::GetAllProjects($login, $active, $order, $description, $filterStartDate,
+        $filterEndDate, $activation, $areaId, $type, $cname, $returnExtendedInfo);
     $string = "<projects>";
 
-    foreach((array) $projects as $project)
-    {
 
-        $string = $string . "<project><id>{$project->getId()}</id><areaId>{$project->getAreaId()}</areaId><customerId>{$project->getCustomerId()}</customerId><activation>{$project->getActivation()}</activation><description>" . escape_string($project->getDescription()) . "</description><customerName>" . escape_string($project->getCustomerName()) . "</customerName><invoice>{$project->getInvoice()}</invoice>";
+    foreach ((array) $projects as $project) {
+        $string = $string . "<project><id>{$project->getId()}</id><areaId>{$project->getAreaId()}</areaId><customerId>{$project->getCustomerId()}</customerId><activation>{$project->getActivation()}</activation><description>" . escape_string( $project->getDescription() ) . "</description><customerName>" . escape_string( $project->getCustomerName() ) . "</customerName><invoice>{$project->getInvoice()}</invoice>";
 
-        if (!is_null($project->getInit()))
+        if ( !is_null( $project->getInit())) {
             $string = $string . "<init format='Y-m-d'>{$project->getInit()->format("Y-m-d")}</init>";
-        else $string = $string . "<init/>";
+        } else {
+            $string = $string . "<init/>";
+        }
 
-        if (!is_null($project->getEnd()))
+        if ( !is_null( $project->getEnd() ) ) {
```

Comment:
Watch out for undesired style changes (probably made by the editor). These add noise to the patch and make it harder to read.